doc/source/deployment.rst

```diff
@@ -38,13 +38,21 @@ Deployment of the complete system in a single machine/VM.
 
 #. Generate the SSL certificates if you do not already have them. The
    resulting certificates will have names test.* because they are
-   self-signed and are not supposed to be used for production.  Once
-   created, copy them to the jupyterhub directory::
+   self-signed and are not supposed to be used for production.
+   You must choose and set a password of your liking, and use it when prompted.
+   Additional information will also be requested, but are not strictly required
+   and can be left as defaults.
+
+   Once created the certificates, copy them to the jupyterhub directory::
 
      cd ../scripts
      sh generate_certificate.sh
      cp test.* ../jupyterhub/
 
+#. Create the database. By default, this is a sqlite file::
+
+     python dbutil.py --db=~/remoteappmanager.db init
```

Comment:
as the console script is available, this is cooler:
`remoteappdb --db=~/remoteappmanager.db init`